GH Actions failing on docs test

**Describe the bug**

When running the `make docs` command, we encounter the following error:

```
/Users/osilkin/Programming/labrador/instructlab/docs/ci.md:37: WARNING: 'myst' cross-reference target not found: 'triggering-an-e2e-job-via-github-web-ui' [myst.xref_missing]
writing output... [100%] release-strategy
generating indices... genindex done
writing additional pages... search done
copying images... [100%] images/github-actions-run-workflow-example.png
dumping search index in English (code: en)... done
dumping object inventory... done
build finished with problems, 1 warning (with warnings treated as errors).
docs: exit 1 (2.37 seconds) /Users/osilkin/Programming/labrador/instructlab> sphinx-build -M html docs docs/build -j auto --keep-going --fail-on-warning --fresh-env -n pid=20049
  docs: FAIL code 1 (4.35=setup[1.98]+cmd[2.37] seconds)
  evaluation failed :( (4.42 seconds)
make: *** [docs] Error 1
```

**To Reproduce**
Steps to reproduce the behavior:
1. Create a new branch based on the current main: https://github.com/instructlab/instructlab/commit/20bd8e839d38be96af4d93f75b2a5169bc34ae5d 
2. Run `make docs` or submit a PR 


**Expected behavior**
They should pass successfully

--- a/docs/conf.py
+++ b/docs/conf.py
@@ -62,6 +62,8 @@ nitpick_ignore = [
     ("py:class", "FieldInfo"),
 ]
 
+myst_heading_anchors = 3
+
 # -- Options for HTML output -------------------------------------------------
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-html-output
 

